ui/lib/core/addon/components/key-value-header.js

```diff
@@ -1,87 +1,69 @@
-import { computed } from '@ember/object';
-import Component from '@ember/component';
+import Component from '@glimmer/component';
 import utils from 'vault/lib/key-utils';
-import layout from '../templates/components/key-value-header';
 import { encodePath } from 'vault/utils/path-encoding-helpers';
 
-export default Component.extend({
-  layout,
-  tagName: 'nav',
-  classNames: 'key-value-header breadcrumb',
-  ariaLabel: 'breadcrumbs',
-  attributeBindings: ['ariaLabel:aria-label', 'aria-hidden'],
+/**
+ * @module KeyValueHeader
+ * KeyValueHeader components show breadcrumbs.
+ *
+ * @example
+ * ```js
+ * <KeyValueHeader @requiredParam={requiredParam} @optionalParam={optionalParam} @param1={{param1}}/>
```

Comment:
Pulling a real example from somewhere in the code might be a little more explanatory. It'd be helpful for me at least to see an example of what this looks like in practice!

Reply:
Ah, yep, I forgot this bit. Copy and pasted.